chain/client/tests/process_blocks.rs

```diff
@@ -2408,6 +2410,51 @@ fn test_not_broadcast_block_on_accept() {
     assert!(network_adapter.requests.read().unwrap().is_empty());
 }
 
+#[test]
+#[should_panic]
+// TODO: reject header version downgrade
```

Comment:
are you going to do this in a separate PR? If so, it is better to add the issue number here